File: Python/String-1/combo_string.py
# problem: https://codingbat.com/prob/p194053
# Given 2 strings, a and b, return a string of the form short+long+short.
# The strings will not be the same length, but they may be empty (length 0).

def combo_string(a, b):
  if len(a) > len(b):
    return b + a + b
  
  return a + b + a

File: Java/String-1/comboString.java
/*
 * problem: https://codingbat.com/prob/p168564
 * Given 2 strings, a and b, return a string of the form short+long+short.
 */

public String comboString(String a, String b) {
  if (a.length() < b.length()) {
    return a + b + a;
  } else {
    return b + a + b;
  }
}
